addons/website_slides/static/src/js/slides_course_prerequisite.js

```diff
@@ -0,0 +1,25 @@
+/** @odoo-module **/
+
+import { qweb } from 'web.core';
+import publicWidget from 'web.public.widget';
+
+publicWidget.registry.websiteSlidesCoursePrerequisite = publicWidget.Widget.extend({
+    selector: '.o_wslides_js_prerequisite_course',
+
+    start() {
+        return this._super(...arguments).then(async () => {
```

Comment:
You could use the "await" syntax.

One indentation level gained :partying_face: